### DISCUSSION TITLE
Adjusting or turning off Ownership Timer

### DISCUSSION THREAD

**Discussion by acidicp** [trickerer/Trinity-Bots#818] [Q&A] [answered]
I looked through all of the commands and could not find one to adjust or disable the ownership timer. In the .patch code I see many instances of "CheckOwershipTimer" and "CheckMasterTimer" but I am not smart enough to know how to adjust those to change or remove the timer. Is there something I could easily do to remove or extend the timer?

**Comment by trickerer** [comment, marked as answer]
Ownership timer can only be managed through config, there is no command interface for that. If you set it to 0 and then reload the config (`.npcbot reload`) bots already running the timer won't expire.

**Reply by acidicp** [reply]
Thanks so much for the quick reply and for building this whole thing!
